javax.xml.stream.FactoryConfigurationError occurs if auth_method is web_identity_token

First of all, thanks for your massive effort in implementing many useful Embulk plugins!

## Summary
When I run Embulk with `auth_method: web_identity_token` in an EKS cluster that successfully setup IAM roles for service accounts as [described in this document](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts-technical-overview.html#pod-configuration),

> Error: javax.xml.stream.FactoryConfigurationError: Provider com.bea.xml.stream.MXParserFactory not found

occurs.

<details><summary>Full error log</summary>

```
...
2020-10-02 12:50:39.946 +0000 [INFO] (0001:transaction): Create a new table: my_database.my_table
2020-10-02 12:50:40.692 +0000 [INFO] (0001:cleanup): Created: s3://my-s3-bucket/path_prefix/000.00.parquet, version_id: null, etag: 2a6da5742db8c870af29d0182563578e-12
org.embulk.exec.PartialExecutionException: javax.xml.stream.FactoryConfigurationError: Provider com.bea.xml.stream.MXParserFactory not found
        at org.embulk.exec.BulkLoader$LoaderState.buildPartialExecuteException(BulkLoader.java:340)
        at org.embulk.exec.BulkLoader.doRun(BulkLoader.java:566)
        at org.embulk.exec.BulkLoader.access$000(BulkLoader.java:35)
        at org.embulk.exec.BulkLoader$1.run(BulkLoader.java:353)
        at org.embulk.exec.BulkLoader$1.run(BulkLoader.java:350)
        at org.embulk.spi.Exec.doWith(Exec.java:22)
        at org.embulk.exec.BulkLoader.run(BulkLoader.java:350)
        at org.embulk.EmbulkEmbed.run(EmbulkEmbed.java:242)
        at org.embulk.EmbulkRunner.runInternal(EmbulkRunner.java:291)
        at org.embulk.EmbulkRunner.run(EmbulkRunner.java:155)
        at org.embulk.cli.EmbulkRun.runSubcommand(EmbulkRun.java:431)
        at org.embulk.cli.EmbulkRun.run(EmbulkRun.java:90)
        at org.embulk.cli.Main.main(Main.java:64)
Caused by: javax.xml.stream.FactoryConfigurationError: Provider com.bea.xml.stream.MXParserFactory not found
        at javax.xml.stream.FactoryFinder.newInstance(FactoryFinder.java:72)
        at javax.xml.stream.FactoryFinder.find(FactoryFinder.java:176)
        at javax.xml.stream.FactoryFinder.find(FactoryFinder.java:92)
        at javax.xml.stream.XMLInputFactory.newInstance(XMLInputFactory.java:136)
        at com.amazonaws.util.XmlUtils.createXmlInputFactory(XmlUtils.java:63)
        at com.amazonaws.util.XmlUtils.access$000(XmlUtils.java:27)
        at com.amazonaws.util.XmlUtils$1.initialValue(XmlUtils.java:36)
        at com.amazonaws.util.XmlUtils$1.initialValue(XmlUtils.java:33)
        at java.lang.ThreadLocal.setInitialValue(ThreadLocal.java:180)
        at java.lang.ThreadLocal.get(ThreadLocal.java:170)
        at com.amazonaws.util.XmlUtils.getXmlInputFactory(XmlUtils.java:54)
        at com.amazonaws.http.StaxResponseHandler.handle(StaxResponseHandler.java:94)
        at com.amazonaws.http.StaxResponseHandler.handle(StaxResponseHandler.java:42)
        at com.amazonaws.http.response.AwsResponseHandlerAdapter.handle(AwsResponseHandlerAdapter.java:69)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutor.handleResponse(AmazonHttpClient.java:1726)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutor.handleSuccessResponse(AmazonHttpClient.java:1446)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeOneRequest(AmazonHttpClient.java:1368)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeHelper(AmazonHttpClient.java:1145)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutor.doExecute(AmazonHttpClient.java:802)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeWithTimer(AmazonHttpClient.java:770)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutor.execute(AmazonHttpClient.java:744)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutor.access$500(AmazonHttpClient.java:704)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutionBuilderImpl.execute(AmazonHttpClient.java:686)
        at com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:550)
        at com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:530)
        at com.amazonaws.services.securitytoken.AWSSecurityTokenServiceClient.doInvoke(AWSSecurityTokenServiceClient.java:1719)
        at com.amazonaws.services.securitytoken.AWSSecurityTokenServiceClient.invoke(AWSSecurityTokenServiceClient.java:1686)
        at com.amazonaws.services.securitytoken.AWSSecurityTokenServiceClient.invoke(AWSSecurityTokenServiceClient.java:1675)
        at com.amazonaws.services.securitytoken.AWSSecurityTokenServiceClient.executeAssumeRoleWithWebIdentity(AWSSecurityTokenServiceClient.java:1074)
        at com.amazonaws.services.securitytoken.AWSSecurityTokenServiceClient.assumeRoleWithWebIdentity(AWSSecurityTokenServiceClient.java:1045)
        at com.amazonaws.auth.STSAssumeRoleWithWebIdentitySessionCredentialsProvider.newSession(STSAssumeRoleWithWebIdentitySessionCredentialsProvider.java:149)
        at com.amazonaws.auth.STSAssumeRoleWithWebIdentitySessionCredentialsProvider.access$000(STSAssumeRoleWithWebIdentitySessionCredentialsProvider.java:40)
        at com.amazonaws.auth.STSAssumeRoleWithWebIdentitySessionCredentialsProvider$1.call(STSAssumeRoleWithWebIdentitySessionCredentialsProvider.java:65)
        at com.amazonaws.auth.STSAssumeRoleWithWebIdentitySessionCredentialsProvider$1.call(STSAssumeRoleWithWebIdentitySessionCredentialsProvider.java:62)
        at com.amazonaws.auth.RefreshableTask.refreshValue(RefreshableTask.java:257)
        at com.amazonaws.auth.RefreshableTask.blockingRefresh(RefreshableTask.java:213)
        at com.amazonaws.auth.RefreshableTask.getValue(RefreshableTask.java:154)
        at com.amazonaws.auth.STSAssumeRoleWithWebIdentitySessionCredentialsProvider.getCredentials(STSAssumeRoleWithWebIdentitySessionCredentialsProvider.java:130)
        at com.amazonaws.auth.STSAssumeRoleWithWebIdentitySessionCredentialsProvider.getCredentials(STSAssumeRoleWithWebIdentitySessionCredentialsProvider.java:40)
        at com.amazonaws.auth.profile.internal.securitytoken.STSProfileCredentialsServiceProvider.getCredentials(STSProfileCredentialsServiceProvider.java:71)
        at com.amazonaws.auth.WebIdentityTokenCredentialsProvider.getCredentials(WebIdentityTokenCredentialsProvider.java:72)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutor.getCredentialsFromContext(AmazonHttpClient.java:1257)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutor.runBeforeRequestHandlers(AmazonHttpClient.java:833)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutor.doExecute(AmazonHttpClient.java:783)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeWithTimer(AmazonHttpClient.java:770)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutor.execute(AmazonHttpClient.java:744)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutor.access$500(AmazonHttpClient.java:704)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutionBuilderImpl.execute(AmazonHttpClient.java:686)
        at com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:550)
        at com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:530)
        at com.amazonaws.services.glue.AWSGlueClient.doInvoke(AWSGlueClient.java:8136)
        at com.amazonaws.services.glue.AWSGlueClient.invoke(AWSGlueClient.java:8103)
        at com.amazonaws.services.glue.AWSGlueClient.invoke(AWSGlueClient.java:8092)
        at com.amazonaws.services.glue.AWSGlueClient.executeCreateTable(AWSGlueClient.java:1726)
        at com.amazonaws.services.glue.AWSGlueClient.createTable(AWSGlueClient.java:1697)
        at org.embulk.output.s3_parquet.catalog.CatalogRegistrator.$anonfun$registerNewParquetTable$2(CatalogRegistrator.scala:177)
        at org.embulk.output.s3_parquet.aws.Aws.withGlue(Aws.scala:46)
        at org.embulk.output.s3_parquet.catalog.CatalogRegistrator.registerNewParquetTable(CatalogRegistrator.scala:177)
        at org.embulk.output.s3_parquet.catalog.CatalogRegistrator.run(CatalogRegistrator.scala:109)
        at org.embulk.output.s3_parquet.S3ParquetOutputPlugin.$anonfun$transaction$1(S3ParquetOutputPlugin.scala:50)
        at java.util.Optional.ifPresent(Optional.java:159)
        at org.embulk.output.s3_parquet.S3ParquetOutputPlugin.transaction(S3ParquetOutputPlugin.scala:38)
        at org.embulk.exec.BulkLoader$4$1$1.transaction(BulkLoader.java:521)
        at org.embulk.exec.LocalExecutorPlugin.transaction(LocalExecutorPlugin.java:50)
        at org.embulk.exec.BulkLoader$4$1.run(BulkLoader.java:516)
        at org.embulk.spi.util.Filters$RecursiveControl.transaction(Filters.java:84)
        at org.embulk.spi.util.Filters$RecursiveControl$1.run(Filters.java:80)
        at org.embulk.standards.RenameFilterPlugin.transaction(RenameFilterPlugin.java:81)
        at org.embulk.spi.util.Filters$RecursiveControl.transaction(Filters.java:76)
        at org.embulk.spi.util.Filters$RecursiveControl$1.run(Filters.java:80)
        at org.embulk.filter.expand_json.ExpandJsonFilterPlugin.transaction(ExpandJsonFilterPlugin.java:108)
        at org.embulk.spi.util.Filters$RecursiveControl.transaction(Filters.java:76)
        at org.embulk.spi.util.Filters.transaction(Filters.java:42)
        at org.embulk.exec.BulkLoader$4.run(BulkLoader.java:511)
        at org.embulk.input.jdbc.AbstractJdbcInputPlugin.transaction(AbstractJdbcInputPlugin.java:224)
        at org.embulk.exec.BulkLoader.doRun(BulkLoader.java:507)
        ... 11 more

Error: javax.xml.stream.FactoryConfigurationError: Provider com.bea.xml.stream.MXParserFactory not found
```
</details>

## Environment

This is the Dockerfile. I verified that switching to `openjdk:8` doesn't help (exactly the same error occurs).

```dockerfile
FROM openjdk:8-alpine

ENV EMBULK_VERSION 0.9.23

RUN apk --update add --virtual build-dependencies curl \
    && mkdir /embulk \
    && curl -o /embulk/embulk -L "https://dl.bintray.com/embulk/maven/embulk-$EMBULK_VERSION.jar" \
    && chmod +x /embulk/embulk \
    && apk del build-dependencies

# Install libc6-compat and gcompat for Embulk Plugins to use JNI and snappy
# * https://github.com/jruby/jruby/wiki/JRuby-on-Alpine-Linux
# * https://stackoverflow.com/a/55568352
RUN apk --update add libc6-compat gcompat

RUN /embulk/embulk gem install embulk-input-mysql embulk-output-s3_parquet

COPY ./config.yml.liquid /config.yml.liquid
CMD [ "/embulk/embulk", "run", "/config.yml.liquid" ]
```

config.yml.liquid:

```yml
# ...
out:
  type: s3_parquet
  region: ap-northeast-1
  bucket: my-s3-bucket
  path_prefix: path_prefix
  compression_codec: snappy
  catalog:
    database: my_database
    table: my_table
  auth_method: web_identity_token
  role_arn: {{env.AWS_ROLE_ARN}}
  role_session_name: something
  web_identity_token_file: {{env.AWS_WEB_IDENTITY_TOKEN_FILE}}
```

When I installed `com.bea.xml.stream.MXParserFactory` manually:

```diff
- CMD [ "/embulk/embulk", "run", "/config.yml.liquid" ]
+ RUN curl -o /libraries/stax-1.2.0.jar -L "https://repo.maven.apache.org/maven2/stax/stax/1.2.0/stax-1.2.0.jar"
+ CMD [ "/embulk/embulk", "-C", "/libraries/stax-1.2.0.jar", "run", "/config.yml.liquid" ]
```

then it fails with the following error:

> Error: com.amazonaws.SdkClientException: Unable to unmarshall response (com.bea.xml.stream.MXParserFactory cannot be cast to javax.xml.stream.XMLInputFactory). Response Code: 200, Response Text: OK

<details><summary>Full error log</summary>

```
org.embulk.exec.PartialExecutionException: com.amazonaws.SdkClientException: Unable to unmarshall response (com.bea.xml.stream.MXParserFactory cannot be cast to javax.xml.stream.XMLInputFactory). Response Code: 200, Response Text: OK
        at org.embulk.exec.BulkLoader$LoaderState.buildPartialExecuteException(BulkLoader.java:340)
        at org.embulk.exec.BulkLoader.doRun(BulkLoader.java:566)
        at org.embulk.exec.BulkLoader.access$000(BulkLoader.java:35)
        at org.embulk.exec.BulkLoader$1.run(BulkLoader.java:353)
        at org.embulk.exec.BulkLoader$1.run(BulkLoader.java:350)
        at org.embulk.spi.Exec.doWith(Exec.java:22)
        at org.embulk.exec.BulkLoader.run(BulkLoader.java:350)
        at org.embulk.EmbulkEmbed.run(EmbulkEmbed.java:242)
        at org.embulk.EmbulkRunner.runInternal(EmbulkRunner.java:291)
        at org.embulk.EmbulkRunner.run(EmbulkRunner.java:155)
        at org.embulk.cli.EmbulkRun.runSubcommand(EmbulkRun.java:431)
        at org.embulk.cli.EmbulkRun.run(EmbulkRun.java:90)
        at org.embulk.cli.Main.main(Main.java:64)
Caused by: com.amazonaws.SdkClientException: Unable to unmarshall response (com.bea.xml.stream.MXParserFactory cannot be cast to javax.xml.stream.XMLInputFactory). Response Code: 200, Response Text: OK
        at com.amazonaws.http.AmazonHttpClient$RequestExecutor.handleResponse(AmazonHttpClient.java:1750)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutor.handleSuccessResponse(AmazonHttpClient.java:1446)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeOneRequest(AmazonHttpClient.java:1368)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeHelper(AmazonHttpClient.java:1145)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutor.doExecute(AmazonHttpClient.java:802)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeWithTimer(AmazonHttpClient.java:770)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutor.execute(AmazonHttpClient.java:744)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutor.access$500(AmazonHttpClient.java:704)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutionBuilderImpl.execute(AmazonHttpClient.java:686)
        at com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:550)
        at com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:530)
        at com.amazonaws.services.securitytoken.AWSSecurityTokenServiceClient.doInvoke(AWSSecurityTokenServiceClient.java:1719)
        at com.amazonaws.services.securitytoken.AWSSecurityTokenServiceClient.invoke(AWSSecurityTokenServiceClient.java:1686)
        at com.amazonaws.services.securitytoken.AWSSecurityTokenServiceClient.invoke(AWSSecurityTokenServiceClient.java:1675)
        at com.amazonaws.services.securitytoken.AWSSecurityTokenServiceClient.executeAssumeRoleWithWebIdentity(AWSSecurityTokenServiceClient.java:1074)
        at com.amazonaws.services.securitytoken.AWSSecurityTokenServiceClient.assumeRoleWithWebIdentity(AWSSecurityTokenServiceClient.java:1045)
        at com.amazonaws.auth.STSAssumeRoleWithWebIdentitySessionCredentialsProvider.newSession(STSAssumeRoleWithWebIdentitySessionCredentialsProvider.java:149)
        at com.amazonaws.auth.STSAssumeRoleWithWebIdentitySessionCredentialsProvider.access$000(STSAssumeRoleWithWebIdentitySessionCredentialsProvider.java:40)
        at com.amazonaws.auth.STSAssumeRoleWithWebIdentitySessionCredentialsProvider$1.call(STSAssumeRoleWithWebIdentitySessionCredentialsProvider.java:65)
        at com.amazonaws.auth.STSAssumeRoleWithWebIdentitySessionCredentialsProvider$1.call(STSAssumeRoleWithWebIdentitySessionCredentialsProvider.java:62)
        at com.amazonaws.auth.RefreshableTask.refreshValue(RefreshableTask.java:257)
        at com.amazonaws.auth.RefreshableTask.blockingRefresh(RefreshableTask.java:213)
        at com.amazonaws.auth.RefreshableTask.getValue(RefreshableTask.java:154)
        at com.amazonaws.auth.STSAssumeRoleWithWebIdentitySessionCredentialsProvider.getCredentials(STSAssumeRoleWithWebIdentitySessionCredentialsProvider.java:130)
        at com.amazonaws.auth.STSAssumeRoleWithWebIdentitySessionCredentialsProvider.getCredentials(STSAssumeRoleWithWebIdentitySessionCredentialsProvider.java:40)
        at com.amazonaws.auth.profile.internal.securitytoken.STSProfileCredentialsServiceProvider.getCredentials(STSProfileCredentialsServiceProvider.java:71)
        at com.amazonaws.auth.WebIdentityTokenCredentialsProvider.getCredentials(WebIdentityTokenCredentialsProvider.java:72)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutor.getCredentialsFromContext(AmazonHttpClient.java:1257)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutor.runBeforeRequestHandlers(AmazonHttpClient.java:833)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutor.doExecute(AmazonHttpClient.java:783)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeWithTimer(AmazonHttpClient.java:770)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutor.execute(AmazonHttpClient.java:744)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutor.access$500(AmazonHttpClient.java:704)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutionBuilderImpl.execute(AmazonHttpClient.java:686)
        at com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:550)
        at com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:530)
        at com.amazonaws.services.glue.AWSGlueClient.doInvoke(AWSGlueClient.java:8136)
        at com.amazonaws.services.glue.AWSGlueClient.invoke(AWSGlueClient.java:8103)
        at com.amazonaws.services.glue.AWSGlueClient.invoke(AWSGlueClient.java:8092)
        at com.amazonaws.services.glue.AWSGlueClient.executeCreateTable(AWSGlueClient.java:1726)
        at com.amazonaws.services.glue.AWSGlueClient.createTable(AWSGlueClient.java:1697)
        at org.embulk.output.s3_parquet.catalog.CatalogRegistrator.$anonfun$registerNewParquetTable$2(CatalogRegistrator.scala:177)
        at org.embulk.output.s3_parquet.aws.Aws.withGlue(Aws.scala:46)
        at org.embulk.output.s3_parquet.catalog.CatalogRegistrator.registerNewParquetTable(CatalogRegistrator.scala:177)
        at org.embulk.output.s3_parquet.catalog.CatalogRegistrator.run(CatalogRegistrator.scala:109)
        at org.embulk.output.s3_parquet.S3ParquetOutputPlugin.$anonfun$transaction$1(S3ParquetOutputPlugin.scala:50)
        at java.util.Optional.ifPresent(Optional.java:159)
        at org.embulk.output.s3_parquet.S3ParquetOutputPlugin.transaction(S3ParquetOutputPlugin.scala:38)
        at org.embulk.exec.BulkLoader$4$1$1.transaction(BulkLoader.java:521)
        at org.embulk.exec.LocalExecutorPlugin.transaction(LocalExecutorPlugin.java:50)
        at org.embulk.exec.BulkLoader$4$1.run(BulkLoader.java:516)
        at org.embulk.spi.util.Filters$RecursiveControl.transaction(Filters.java:84)
        at org.embulk.spi.util.Filters$RecursiveControl$1.run(Filters.java:80)
        at org.embulk.standards.RenameFilterPlugin.transaction(RenameFilterPlugin.java:81)
        at org.embulk.spi.util.Filters$RecursiveControl.transaction(Filters.java:76)
        at org.embulk.spi.util.Filters$RecursiveControl$1.run(Filters.java:80)
        at org.embulk.filter.expand_json.ExpandJsonFilterPlugin.transaction(ExpandJsonFilterPlugin.java:108)
        at org.embulk.spi.util.Filters$RecursiveControl.transaction(Filters.java:76)
        at org.embulk.spi.util.Filters.transaction(Filters.java:42)
        at org.embulk.exec.BulkLoader$4.run(BulkLoader.java:511)
        at org.embulk.input.jdbc.AbstractJdbcInputPlugin.transaction(AbstractJdbcInputPlugin.java:224)
        at org.embulk.exec.BulkLoader.doRun(BulkLoader.java:507)
        ... 11 more
Caused by: java.lang.ClassCastException: com.bea.xml.stream.MXParserFactory cannot be cast to javax.xml.stream.XMLInputFactory
        at javax.xml.stream.XMLInputFactory.newInstance(XMLInputFactory.java:136)
        at com.amazonaws.util.XmlUtils.createXmlInputFactory(XmlUtils.java:63)
        at com.amazonaws.util.XmlUtils.access$000(XmlUtils.java:27)
        at com.amazonaws.util.XmlUtils$1.initialValue(XmlUtils.java:36)
        at com.amazonaws.util.XmlUtils$1.initialValue(XmlUtils.java:33)
        at java.lang.ThreadLocal.setInitialValue(ThreadLocal.java:180)
        at java.lang.ThreadLocal.get(ThreadLocal.java:170)
        at com.amazonaws.util.XmlUtils.getXmlInputFactory(XmlUtils.java:54)
        at com.amazonaws.http.StaxResponseHandler.handle(StaxResponseHandler.java:94)
        at com.amazonaws.http.StaxResponseHandler.handle(StaxResponseHandler.java:42)
        at com.amazonaws.http.response.AwsResponseHandlerAdapter.handle(AwsResponseHandlerAdapter.java:69)
        at com.amazonaws.http.AmazonHttpClient$RequestExecutor.handleResponse(AmazonHttpClient.java:1726)
        ... 72 more

Error: com.amazonaws.SdkClientException: Unable to unmarshall response (com.bea.xml.stream.MXParserFactory cannot be cast to javax.xml.stream.XMLInputFactory). Response Code: 200, Response Text: OK
```
</details>

Am I missing something important? Any help/recommendations are welcome 🙏  (Note that I'm not familiar with Java 😞)

--- a/src/main/scala/org/embulk/output/s3_parquet/S3ParquetOutputPlugin.scala
+++ b/src/main/scala/org/embulk/output/s3_parquet/S3ParquetOutputPlugin.scala
@@ -47,7 +47,9 @@ class S3ParquetOutputPlugin extends OutputPlugin {
         defaultGlueTypes =
           support.parquetSchema.transform((k, v) => v.glueDataType(k))
       )
-      cr.run()
+      ContextClassLoaderSwapper.usingPluginClass {
+        cr.run()
+      }
     }
 
     Exec.newConfigDiff

